portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.data;
+
+import java.util.Date;
+
+public class Comment {
+    public Comment() {
+
+    }
+
+    public Comment(long id, String name, String description, long postDate) {
```

Comment:
Same as previous this constructor should not be public

Reply:
Done